todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsContract.java

```diff
@@ -35,7 +34,7 @@
         boolean isActive();
     }
 
-    interface Presenter extends BasePresenter {
-
+    interface Presenter {
+        void startStatisticsPresenter();
```

Comment:
better naming here would be nice, this is later calling `loadStatistics()`. How about changing this to `presenter.loadStatistics()`?